src/main/java/seedu/address/logic/commands/CategorizeTagCommand.java

```diff
@@ -21,7 +21,7 @@ public class CategorizeTagCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Categorizes a tag. Changes all occurrences of the specified tag to the desired category.\n"
-            + "Parameters: " + PREFIX_TAG + "TAG (existing tag label) CATEGORY\n"
+            + "Parameters: " + PREFIX_TAG + "TAG (at least one existing tag label) CATEGORY\n"
```

Comment:
I think this would imply “t/tag1 tag2 tag3…” when in reality the correct format is “t/tag1 t/tag2 ….”

Reply:
> I think this would imply “t/tag1 tag2 tag3…” when in reality the correct format is “t/tag1 t/tag2 ….”

That make sense